src/screens/ProfileScreen/elements/BottomSheetRealname.js

```diff
@@ -1,16 +1,10 @@
 import * as React from 'react';
-import {
-  View,
-  Text,
-  StyleSheet,
-  TextInput,
-  ActivityIndicator,
-} from 'react-native';
+import {View, Text, StyleSheet, TextInput, ActivityIndicator} from 'react-native';
 
 import {Button} from '../../../components/Button';
 import {BottomSheet} from '../../../components/BottomSheet';
 import {fonts} from '../../../utils/fonts';
-import {colors} from '../../../utils/colors';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use the new `COLORS` object from the `theme` module instead of the old `colors` object. This change is consistent with the PR's goal of standardizing color usage across the application.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->